src/javascripts/components/Dropdown.jsx

```diff
@@ -1,12 +1,92 @@
 const React = require('react');
-// const ReactDOM = require('react-dom');
+var classNames = require('classnames');
 
 export default class Dropdown extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isOpen: false
+    };
+
+    this._switchState = this._switchState.bind(this);
+    this._loseFocus = this._loseFocus.bind(this);
```

Comment:
can we change the name of this function to something like _closeDropdown since that is what it does, that way if we need to reuse it it will make more sense